frontend/public/locales/fr/gcweb.json

```diff
@@ -112,7 +112,7 @@
   },
   "error-summary": {
     "header_one": "L'erreur suivante a été trouvée dans le formulaire\u00a0:",
-    "header_other": "Les {{count}} erreurs suivantes ont été trouvées dans le formulaire\u00a0:"
+    "header_other": "Les {{count}} erreurs suivantes ont été trouvées dans le formulaire:"
```

Comment:
The non-breaking space should be left in before the colon in my opinion. The ADO doesn't seem to reference a Figma - I think we should make the argument to leave this as is.

Reply:
Good call. French always have a space before colon.

Reply:
Re-added space